CMakeLists.txt

```diff
@@ -126,7 +126,10 @@ if (NOT CMAKE_GENERATOR MATCHES "Xcode|Visual Studio")
 endif ()
 message(STATUS "")
 
-find_package(Git QUIET) # no need for this msg. It comes from cmake.findgit()
+################################################################################
+# VCS info
+################################################################################
+find_package(Git REQUIRED)
```

Comment:
what happens if someone downloads the code from GitHub as a zip-file? Does it compile without git?

Reply:
Woops... :) will be undone